### DISCUSSION TITLE
Learning Tidyverse - Dplyr + Tidyr (spanish)

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/data-science#33] [Learning Resources]
YouTube tutorial:  https://www.youtube.com/watch?v=7f6f0zw2XDQ&ab_channel=RLadiesMexico 

Please comment below how help full this video was:

give a rating out 5 for learning github desktop
what stuck out and you learned
what was confusing
general thoughts

**Comment by AnaPeraltaG** [comment]
Hi @ran-codes,

- **Rating**
        5/5 rating
- **What stuck out and I learned:**
      - Learned how to use lots of functions from different tidyverse packages. 
      - Loved that they shared this site:  https://www.tidyverse.org  (I downloaded useful cheat sheets from there)
      - They guide you on how to do exercises in R and provide their Github repository for you to clone and follow along 
- **What was confusing:**
      - It was all very clear and easy to follow
- **General thoughts:**
      - I liked their video. Easy to follow, showed useful functions. 
      - The video is a bit long (1 hr 43 min). However, they divided the training in 2 portions. The first one (introduction and first exercises in R) ends in minute 37. The second part of the training (additional exercises showing how to read in different types of files and modify the datasets) takes the rest of the time.

Thanks for finding and sharing this! :)
